src/core/manifest.ts

```diff
@@ -47,7 +47,7 @@ class ManifestService extends NodeService<Stack> {
    * @param  {JSON} manifest manifest JSON
    * @returns Promise with corresponding transaction id
    */
-  public async generate(vaultId: string, manifest?: JSON | Object): Promise<{ transactionId: string, object: Stack }> {
+  public async generate(vaultId: string, indexName?: string, manifest?: JSON | Object): Promise<{ transactionId: string, object: Stack }> {
```

Comment:
can you reverse it?
`generate(vaultId: string, manifest?: JSON | Object, indexName?: string)`
that way its backward compatible